libs/core/src/lib/tabs/tab-link/tab-link.directive.ts

```diff
@@ -16,7 +16,7 @@ import { AbstractFdNgxClass } from '@fundamental-ngx/core/utils';
     selector: '[fd-tab-link]',
     host: {
         role: 'tab',
-        tabindex: '0'
+        tabindex: '-1'
```

Comment:
After this change tabbing though tabs is not possible anymore.

Reply:
Yes that's the requirement. here we have to tab using left and right arrow key. Please refer below link for more details.
https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html